index.js

```diff
@@ -166,6 +149,10 @@ module.exports = function () {
       'dropped6': Buffer.alloc(0)
     })
   }
+}
+
+utPex.prototype.name = 'ut_pex'
 
+module.exports = () => {
   return utPex
```

Comment:
I think this should be `return new utPex()`

Reply:
not according to how bittorrent-protocol handle it

https://github.com/webtorrent/bittorrent-protocol/blob/c23fba4203073cd02c7ce2b02992394ada6afd79/index.js#L135-L147

bittorrent-protocol is calling `new utPex` for you

return utPex is what it did before and that is what it's doing now also

Reply:
Ah, right, then we should export the class itself. Maybe also stop using prototype by assigning a name property

Reply:
@KayleePop and I already discussed exporting the class itself https://github.com/webtorrent/ut_pex/pull/23#discussion_r213355744 

if we do that it will break webtorrent that dose `wire.use(utPex())`

Reply:
> Maybe also stop using prototype by assigning a name property

Disagree...